akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala

```diff
@@ -1367,6 +1367,8 @@ private[akka] class WriteAggregator(
     case ReceiveTimeout ⇒ reply(ok = false)
   }
 
+  def senderAddress(): Address = sender().path.address
```

Comment:
needed for the new test `WriteAggregatorSpec`, which is not really related to the cache (I just increased the tests coverage a bit)